src/vs/workbench/browser/parts/panel/media/panelPart.css

```diff
@@ -0,0 +1,13 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.monaco-workbench > .panel > .content > .panel {
+	height: 100%;
+}
+
+.monaco-workbench > .part.panel > .title {
+	display: block;
+	border-top: 1px solid #007ACC
```

Comment:
@isidorn is this value not dynamic depending on debug, no workspace or workspace? I would check what the statusbar does to set this color.